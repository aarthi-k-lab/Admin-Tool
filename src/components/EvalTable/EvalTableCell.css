.redText {
  color: red;
}

.blackText {
  color: black;
}

.redText,
.blackText {
  text-decoration: none !important;
}
