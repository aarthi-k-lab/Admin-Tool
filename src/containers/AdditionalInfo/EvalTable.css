.container {
    max-height: 653;
    border-bottom: 0 !important;
}

.tableStyle {
    width: auto;
    table-layout: auto;
}

.head {
    max-height: 10;
    background: #f7f7f7;
}

.headStyle {
    color: grey !important;
    text-align: center !important;
}

.footer {
    height: 10;
    max-height: 0;
    overflow: unset;
}

.footer>td>div {
    height: 10;
    min-height: 10;
}