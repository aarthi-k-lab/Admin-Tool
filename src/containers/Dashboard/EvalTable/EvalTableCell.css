.redText {
  color: red;
}

.blackText {
  color: black;
}

.redText,
.disabled,
.blackText {
  text-decoration: none !important;
}

.pointer {
  cursor: pointer !important;
}

.loanActivityLink {
  cursor: pointer;
  color: blue;
  text-decoration: none;
}
