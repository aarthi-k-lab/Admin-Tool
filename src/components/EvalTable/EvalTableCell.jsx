import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import './EvalTableCell.css';

const EvalTableCell = ({ value, styleProps, addLink }) => {
  const linkedSpan = (
    <Link styleName={styleProps} to="/frontend-evaluation">
      <span>{ value }</span>
    </Link>
  );
  const span = <span>{ value }</span>;
  if (addLink) {
    return linkedSpan;
  }
  return span;
};

EvalTableCell.PropTypes = {
  value: PropTypes.string.isRequired,
  styleProps: PropTypes.string.isRequired,
  addLink: PropTypes.bool.isRequired,
};

export default EvalTableCell;
