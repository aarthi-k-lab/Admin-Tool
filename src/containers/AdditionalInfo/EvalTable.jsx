import React from 'react';
import PropTypes from 'prop-types';
import Table from '@material-ui/core/Table';
import TableBody from '@material-ui/core/TableBody';
import TableCell from '@material-ui/core/TableCell';
import TableContainer from '@material-ui/core/TableContainer';
import TableHead from '@material-ui/core/TableHead';
import TableRow from '@material-ui/core/TableRow';
import Paper from '@material-ui/core/Paper';
import * as R from 'ramda';
import Row from './Row';
import './EvalTable.css';

const headers = [
  {
    colSpan: 1,
    name: 'EVAL ID',
  },
  {
    colSpan: 1,
    name: 'RESOLUTION ID',
  },
  {
    colSpan: 1,
    name: 'STATUS',
  },
  {
    colSpan: 2,
    name: 'STATUS DATE',
  },
  {
    colSpan: 1,
    name: 'SUBSTATUS',
  },
  {
    colSpan: 2,
    name: 'SUBSTATUS DATE',
  },
  {
    colSpan: 1,
    name: 'HISTORY',
  },
];

const EvalTable = ({ rows, selectRow, selectedIndex }) => {
  const rowsPerPage = 15;
  const emptyRows = rowsPerPage - Math.min(rowsPerPage, rows.length);
  if (!R.isEmpty(rows)) {
    return (
      <TableContainer component={Paper} styleName="container">
        <Table aria-label="collapsible table" fixedHeader={false} size="small" styleName="tableStyle">
          <TableHead styleName="head">
            <TableRow>
              {headers.map(header => (<TableCell key={header.name} align="center" colSpan={header.colSpan} styleName="headStyle">{header.name}</TableCell>))}
            </TableRow>
          </TableHead>
          <TableBody>
            {rows.map((row, idx) => (
              <Row
                key={`${row.resolutionId}-${row.statusDate}`}
                changeColor={selectedIndex === idx}
                data={row}
                onClick={() => selectRow(idx, row.evalId)}
              />
            ))}
            {emptyRows > 0 && (
              <TableRow>
                <TableCell colSpan={6} styleName="headStyle" />
              </TableRow>
            )}
          </TableBody>
        </Table>
      </TableContainer>
    );
  }

  return null;
};

EvalTable.defaultProps = {
  rows: [],
};

EvalTable.propTypes = {
  rows: PropTypes.arrayOf(PropTypes.shape({
    evalId: PropTypes.string,
    loanNumber: PropTypes.string.isRequired,
  })),
  selectedIndex: PropTypes.number.isRequired,
  selectRow: PropTypes.func.isRequired,
};


export default EvalTable;
