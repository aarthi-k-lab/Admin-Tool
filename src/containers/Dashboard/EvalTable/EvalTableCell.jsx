import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import { connect } from 'react-redux';
// import Replay from '@material-ui/icons/Replay';
import Tooltip from '@material-ui/core/Tooltip';
import IconButton from '@material-ui/core/IconButton';
import { selectors as loginSelectors } from 'ducks/login';
import RouteAccess from 'lib/RouteAccess';
import './EvalTableCell.css';
// import styles from './EvalTableCell.css';

class EvalTableCell extends React.PureComponent {
  constructor(props) {
    super(props);
    this.handleLinkClick = this.handleLinkClick.bind(this);
    this.route = this.route.bind(this);
  }

  handleLinkClick() {
    const { click } = this.props;
    click();
  }

  route() {
    const { user } = this.props;
    const groups = user && user.groupList;
    if (RouteAccess.hasLoanActivityAccess(groups)) {
      return '/loan-activity';
    }
    return '/';
  }

  render() {
    const { value, styleProps } = this.props;
    let renderCellValue = '';
    switch (value) {
      case 'Loan Activity':
        renderCellValue = (
          <Link
            onClick={this.handleLinkClick}
            styleName="loanActivityLink"
            to={this.route()}
          >
            {value}
          </Link>
        );
        break;
      case 'Reject':
        renderCellValue = (
          <Tooltip
            styleName="tooltip"
            title="UnReject"
          >
            <IconButton onClick={this.handleLinkClick} styleName="reject-icon">
              <img alt="UnReject" src="/static/img/Revoke.svg" />
            </IconButton>
          </Tooltip>
        );
        break;
      default:
        renderCellValue = (
          <span styleName={styleProps}>
            {value}
          </span>
        );
    }
    return (
      <div>
        {renderCellValue}
      </div>
    );
  }
}

EvalTableCell.propTypes = {
  click: PropTypes.func.isRequired,
  styleProps: PropTypes.string.isRequired,
  user: PropTypes.shape({
    groupList: PropTypes.array,
    skills: PropTypes.objectOf(PropTypes.array).isRequired,
    userDetails: PropTypes.shape({
      email: PropTypes.string,
      jobTitle: PropTypes.string,
      name: PropTypes.string,
    }),
    userGroups: PropTypes.array,
  }).isRequired,
  value: PropTypes.string.isRequired,
};

const mapStateToProps = state => ({
  user: loginSelectors.getUser(state),
});

export default connect(mapStateToProps, null)(EvalTableCell);
