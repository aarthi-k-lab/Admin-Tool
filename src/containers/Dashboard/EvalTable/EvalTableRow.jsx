import React from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import EvalTableCell from './EvalTableCell';
import { operations, selectors } from '../../../state/ducks/dashboard';

class EvalTableRow extends React.PureComponent {
  constructor(props) {
    super(props);
    this.handleLinkClick = this.handleLinkClick.bind(this);
  }

  handleLinkClick() {
    const { row, searchLoanResult } = this.props;
    const { loanNumber } = searchLoanResult;
    const payLoad = { loanNumber, ...row.original };
    const { onSelectEval } = this.props;
    onSelectEval(payLoad);
  }

  render() {
    const getStyles = (row) => {
      let styles = '';
      if (row.original.tstatus === 'Paused') {
        styles = 'blackText';
      } else if (!row.original.assignee && row.column.Header === 'ASSIGNED TO') {
        styles = 'redText pointer';
      } else if (row.original.assignee && (row.original.assignee === 'In Queue' || row.original.assignee === 'N/A')) {
        styles = 'blackText';
      } else {
        styles = 'blackText pointer';
      }
      return styles;
    };
    const { row } = this.props;
    let cellData = null;
    switch (row.column.Header) {
      case 'ASSIGNED TO':
        cellData = <EvalTableCell styleProps={getStyles(row)} value={row.value ? row.value : 'Unassigned'} />;
        break;
      case 'ACTIONS':
        cellData = (
          <EvalTableCell
            click={() => this.handleLinkClick()}
            styleProps={getStyles(row)}
            value="Loan Activity"
          />
        );
        break;
      default:
        cellData = <EvalTableCell styleProps={getStyles(row)} value={row.value} />;
    }
    return (
      <div>
        {cellData}
      </div>
    );
  }
}

EvalTableRow.propTypes = {
  onSelectEval: PropTypes.func.isRequired,
  row: PropTypes.arrayOf(PropTypes.shape({
    evalId: PropTypes.string.isRequired,
  })).isRequired,
  searchLoanResult: PropTypes.arrayOf(PropTypes.shape({
    loanNumber: PropTypes.string.isRequired,
    valid: PropTypes.bool,
  })).isRequired,
};
const mapDispatchToProps = dispatch => ({
  onSelectEval: operations.onSelectEval(dispatch),
});

const mapStateToProps = state => ({
  searchLoanResult: selectors.searchLoanResult(state),
});

const EvalTableRowContainer = connect(
  mapStateToProps,
  mapDispatchToProps,
)(EvalTableRow);

export default EvalTableRowContainer;
