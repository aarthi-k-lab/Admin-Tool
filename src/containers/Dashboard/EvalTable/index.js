import EvalTableRow from './EvalTableRow';
import EvalTableCell from './EvalTableCell';

export {
  EvalTableRow,
  EvalTableCell,
};
