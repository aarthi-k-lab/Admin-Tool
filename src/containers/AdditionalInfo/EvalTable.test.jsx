import React from 'react';
import renderer from 'react-test-renderer';
import EvalTable from './EvalTable';

const selectRow = jest.fn();
describe('renders <EvalTable /> when selected index is 0', () => {
  const props = {
    selectRow,
    rows: [{
      evalId: 'evalId',
      resolutionId: 'resolutionId',
      statusDate: 'statusDate',
      status: 'status',
      substatus: 'subStatus',
      loanNumber: 'loanNumber',
      substatusDate: 'substatusDate',
      evalHistory: [{
        EvalId: 'EvalId',
        ChangeType: 'ChangeType',
        ApprovalType: 'ApprovalType',
        UpdateDate: 'UpdateDate',
        UserName: 'UserName',
      }],
    },
    {
      evalId: 'evalId1',
      resolutionId: 'resolutionId1',
      statusDate: 'statusDate1',
      status: 'status1',
      substatus: 'subStatus1',
      loanNumber: 'loanNumber1',
      substatusDate: 'substatusDate1',
      evalHistory: [{
        EvalId: 'EvalId1',
        ChangeType: 'ChangeType1',
        ApprovalType: 'ApprovalType1',
        UpdateDate: 'UpdateDate1',
        UserName: 'UserName1',
      }],
    }],
    selectedIndex: 1,
  };
  test('snapshot test', () => {
    const tree = renderer
      .create(<EvalTable {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('renders <EvalTable /> when selected index is 1', () => {
  const props = {
    selectRow,
    rows: [{
      evalId: 'evalId',
      resolutionId: 'resolutionId',
      statusDate: 'statusDate',
      status: 'status',
      substatus: 'subStatus',
      loanNumber: 'loanNumber',
      substatusDate: 'substatusDate',
      evalHistory: [{
        EvalId: 'EvalId',
        ChangeType: 'ChangeType',
        ApprovalType: 'ApprovalType',
        UpdateDate: 'UpdateDate',
        UserName: 'UserName',
      }],
    },
    {
      evalId: 'evalId1',
      resolutionId: 'resolutionId1',
      statusDate: 'statusDate1',
      status: 'status1',
      substatus: 'subStatus1',
      loanNumber: 'loanNumber1',
      substatusDate: 'substatusDate1',
      evalHistory: [{
        EvalId: 'EvalId1',
        ChangeType: 'ChangeType1',
        ApprovalType: 'ApprovalType1',
        UpdateDate: 'UpdateDate1',
        UserName: 'UserName1',
      }],
    }],
    selectedIndex: 1,
  };
  test('snapshot test', () => {
    const tree = renderer
      .create(<EvalTable {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
