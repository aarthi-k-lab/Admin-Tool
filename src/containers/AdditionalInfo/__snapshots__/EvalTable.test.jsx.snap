// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <EvalTable /> when selected index is 0 snapshot test 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root containers-AdditionalInfo-EvalTable__container MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="collapsible table"
    className="MuiTable-root containers-AdditionalInfo-EvalTable__tableStyle"
    fixedHeader={false}
    role={null}
  >
    <thead
      className="MuiTableHead-root containers-AdditionalInfo-EvalTable__head"
      role={null}
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          EVAL ID
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          RESOLUTION ID
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          STATUS
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
          scope="col"
        >
          STATUS DATE
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          SUBSTATUS
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
          scope="col"
        >
          SUBSTATUS DATE
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          HISTORY
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root root "
        onClick={[Function]}
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          role="cell"
          scope="row"
        >
          evalId1
        </th>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          resolutionId1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          status1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          statusDate1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
        >
          subStatus1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          substatusDate1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          onClick={[Function]}
          style={
            Object {
              "color": "lightseagreen",
              "whiteSpace": "nowrap",
            }
          }
        >
          Show History 
        </td>
      </tr>
      <tr
        className="MuiTableRow-root root containers-AdditionalInfo-Row__setBackground"
        onClick={[Function]}
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          role="cell"
          scope="row"
        >
          evalId
        </th>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          resolutionId
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          status
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          statusDate
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
        >
          subStatus
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          substatusDate
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          onClick={[Function]}
          style={
            Object {
              "color": "lightseagreen",
              "whiteSpace": "nowrap",
            }
          }
        >
          Show History 
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        role={null}
        style={
          Object {
            "height": 767,
          }
        }
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-sizeSmall"
          colSpan={6}
        />
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`renders <EvalTable /> when selected index is 1 snapshot test 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root containers-AdditionalInfo-EvalTable__container MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="collapsible table"
    className="MuiTable-root containers-AdditionalInfo-EvalTable__tableStyle"
    fixedHeader={false}
    role={null}
  >
    <thead
      className="MuiTableHead-root containers-AdditionalInfo-EvalTable__head"
      role={null}
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          EVAL ID
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          RESOLUTION ID
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          STATUS
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
          scope="col"
        >
          STATUS DATE
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          SUBSTATUS
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
          scope="col"
        >
          SUBSTATUS DATE
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          scope="col"
        >
          HISTORY
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root root "
        onClick={[Function]}
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          role="cell"
          scope="row"
        >
          evalId1
        </th>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          resolutionId1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          status1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          statusDate1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
        >
          subStatus1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          substatusDate1
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          onClick={[Function]}
          style={
            Object {
              "color": "lightseagreen",
              "whiteSpace": "nowrap",
            }
          }
        >
          Show History 
        </td>
      </tr>
      <tr
        className="MuiTableRow-root root containers-AdditionalInfo-Row__setBackground"
        onClick={[Function]}
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
          role="cell"
          scope="row"
        >
          evalId
        </th>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          resolutionId
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
        >
          status
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          statusDate
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={1}
        >
          subStatus
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          colSpan={2}
        >
          substatusDate
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeSmall"
          onClick={[Function]}
          style={
            Object {
              "color": "lightseagreen",
              "whiteSpace": "nowrap",
            }
          }
        >
          Show History 
        </td>
      </tr>
      <tr
        className="MuiTableRow-root"
        role={null}
        style={
          Object {
            "height": 767,
          }
        }
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body containers-AdditionalInfo-EvalTable__headStyle MuiTableCell-sizeSmall"
          colSpan={6}
        />
      </tr>
    </tbody>
  </table>
</div>
`;
